### ISSUE TITLE
set(x) in parallel to list(x) needed morphology.tex

### ISSUE THREAD

**Issue by stefan11** [langsci/hpsg-handbook#44]
```latex
\avm{
\emph{realisation-rule} \impl
	[mud & \1 set!(msp)! \\
	ms & \1 $\cup$ set!(msp)! \\
	mph & \list*(morph)]
}
```

**Comment by kopeckyf** [comment]
You need to define these commands yourself. `\list` and `\list*` are not shipped with `langsci-avm`, it's a user definition as well. The procedure is explained in §3.3, "Defining input patterns", of the documentation. 

Maybe you want sth along these lines?

```latex
\avmdefinecommand{set}[set]
  {
    attributes = \itshape,
    delimfactor = 900,
    delimfall  = 10pt
  }
```

And then use the starred `\set*` if the set is displayed in a value cell, but `\set` if in an attribute cell.

**Comment by stefan11** [comment]
I did this but this threw an error. It is now fixed by using \avmset rather than \set. I think you have something with \set in the code. Maybe this breaks it.

![grafik](https://user-images.githubusercontent.com/4655859/109612923-bcbbc100-7b30-11eb-8642-724b1cce8774.png)

There is a new problem with spacing of italics text and the brakets. I think the distance is too close.

**Comment by kopeckyf** [comment]
No, the code snippet I posted above works fine in unmodified `langsci-avm` and is compatible with `langscibook`. Maybe the error message comes from somewhere else? It's hard to tell without the complete error message. If this is still a problem, please provide a MWE and the error it produces.

**Comment by stefan11** [comment]
Frank Richter defined set, check this.

**Comment by stefan11** [comment]
This is somehow obsolete now, since I have new commands that just print text without using structured brackets:
```latex
\newcommand{\listOf}[1]{\upshape\textit{list}(\textit{#1})}
\newcommand{\setOf}[1]{\upshape\textit{set}(\textit{#1})}
\newcommand{\posetOf}[1]{\upshape\textit{poset}(\textit{#1})}
```

By the way: \listof was also used within the \avm environment, so I had to go for \listOf.

@kopeckyf you may close this once take over to some other repository.
